### PR TITLE
NVD URL tags and additional definitions

### PR DESCRIPTION
**Pull request by chandanbn**
- Add NVD's URL tags 
- Definitions for optional fields: configurations, workarounds, exploits, timeline, and credits.
- Add pattern for CWE Ids
- Fix a description.

### CODE REVIEW

File: cve_json_schema/v5.x_discuss/cve508_containerized.schema [hunk 1]
```diff
@@ -505,7 +533,8 @@
                                 "CWE-ID": {
                                     "type": "string",
                                     "description": "CWE ID of the CWE that best describes this problem_type entry",
-                                    "minLength": 1
+                                    "minLength": 5,
+                                    "pattern":"^CWE-[0-9]+$"
```
Comment (mprpic) [inline cve_json_schema/v5.x_discuss/cve508_containerized.schema:537]: Technically this allows an invalid `CWE-0`, could use this instead: `"^CWE-[1-9]\\d{0,5}$"`

Comment (chandanbn) [reply cve_json_schema/v5.x_discuss/cve508_containerized.schema:537]: What is the CWE for "No problem"? ;-)

Conceptually, a CNA could add a ADP container with CWE set to CWE-0 to provide an opinion that the CVE is a non-issue.

Comment (mprpic) [reply cve_json_schema/v5.x_discuss/cve508_containerized.schema:537]: I don't think `CWE-0` is the right way to indicate "no security impact" :-) If a CNA really wanted to do that (as opposed to *not* publishing a non-vulnerability in the first place), they could assign it a CVSS score of 0.0.

**Comment by chandanbn** [comment]
Suggested changes are in the master branch. Closing this request.

**Pull request closed without merging**